common-libs/common/package-lock.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "@affinidi/common",
-  "version": "2.0.0-beta.4",
+  "version": "2.0.0-beta.5",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@affinidi/common",
-      "version": "2.0.0-beta.4",
+      "version": "2.0.0-beta.5",
       "license": "ISC",
       "dependencies": {
         "@affinidi/affinity-metrics-lib": "^0.0.25",
+        "@affinidi/tiny-lds-ecdsa-secp256k1-2019": "^1.1.0",
```

Comment:
In order for monorepo to work properly, you should only ever do `npm run bootstrap` from the project root directory, and never use `npm install` inside the specific packages directories

Reply:
reverted the change